XamarinCommunityToolkit/Views/AvatarView/ImageSourceValidator.android.ios.macos.uwp.wpf.gtk.tizen.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+#if MONOANDROID
+using Xamarin.Forms.Platform.Android;
+#elif __IOS__
+using Xamarin.Forms.Platform.iOS;
+#elif __MACOS__
+using Xamarin.Forms.Platform.MacOS;
+#elif UWP
+using Xamarin.Forms.Platform.UWP;
+#elif NET471
+using Xamarin.Forms.Platform.GTK.Renderers;
+#elif TIZEN
+using Xamarin.Forms.Platform.Tizen;
+#else
+using Xamarin.Forms.Platform.WPF;
+#endif
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+#if MONOANDROID || __IOS__ || __MACOS__
```

Comment:
any reason for these `using`s be here (inside the namespace)?

Reply:
Otherwise, we will have to specify the full namespace path